### ISSUE TITLE
Help: Point Cloud has no connectivity

### ISSUE THREAD

**Issue by fmigneault** [abnerrjo/CylinderDetection#1]
@abnerrjo 
Hi, I have tried using your application to apply the "Cylinder Detector", but I get the following error message. 
Due to occlusions, most of my samples are not complete. Is this application to work for samples similar to the one below, or is there something else I need to do to work with it? Thanks

![image](https://user-images.githubusercontent.com/19194484/160943164-346bb0d3-e372-425a-8f61-16ff5e2fd2fb.png)

**Comment by karantai** [comment]
Did you solve the problem?

**Comment by fmigneault** [comment]
@karantai 
No, sadly I have not worked further on this project since then.

**Comment by Bombaninha** [comment]
Did you estimate the normals?
